### PR TITLE
Fix (saving previous before opening new item)

### PR DESCRIPTION
**Pull request by Aca-jov**
SD-1926

### CODE REVIEW

**Comment by plamut** [comment]
tests :wink:

**Comment by ioanpocol** [comment]
@Aca-jov This PR is very old. Can you fix it?

**Comment by ioanpocol** [comment]
Please rebase this PR

**Comment by Aca-jov** [comment]
Rebased and test added

File: client/app/scripts/superdesk-workspace/content/content.js [hunk 1]
```diff
@@ -18,32 +18,53 @@
                 'type'
             ];
 
+            var scope = $scope;
+
             /**
              * Create an item and start editing it
              */
             this.create = function(type) {
+                if (scope && scope.dirty){
```
Comment (mugurrus) [inline client/app/scripts/superdesk-workspace/content/content.js]: It's better to move this conditional in closeOpenNew() then to repeat it in multiple methods.

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**